sencha-workspace/SlateDemonstrationsStudent/app/view/Dashboard.js

```diff
@@ -232,7 +243,8 @@ Ext.define('SlateDemonstrationsStudent.view.Dashboard', {
             me.getCompetenciesSummary(),
             me.getRecentProgress(),
             me.getLegend(),
-            me.getCardsCt()
+            me.getCardsCt(),
+            me.getNonEnrollmentMessage()
```

Comment:
This isn't working how the others are, they all have a transformation in place that turn the config into an instance with some sugar that translates true/false into hidden: false/true. Replicate that for this new "slot component" for consistency and then we can do `me.setNonEnrollmentMessage(!studentEnrolled)`